TunnelVisionLabs.ReferenceAssemblyAnnotator/NullableAttributes.vb

```diff
@@ -110,4 +110,65 @@ Namespace Global.System.Diagnostics.CodeAnalysis
         Public ReadOnly Property ParameterValue As Boolean
     End Class
 
+    ''' <summary>Specifies that the method or property will ensure that the listed field and property members have not-null values.</summary>
+    <AttributeUsage(AttributeTargets.Method Or AttributeTargets.Property, Inherited:=False, AllowMultiple:=True)>
+    Friend NotInheritable Class MemberNotNullAttribute
```

Comment:
I'm by no means a VB.NET expert, so please double check (and forgive any rookie mistake 😅)